.travis.yml

```diff
@@ -287,6 +287,7 @@ before_cache:
 install:
 # Match the version of protobuf being installed via apt
   - pip install --user protobuf==3.1.0
+  - pip install --user timeout-decorator
```

Comment:
That's used by the Event and OlaClient tests in ClientWrapperTest.  Not used outside the tests.  Honestly it could be dropped if it were preferable to let the tests hang if something breaks and let the build system kill them, but I hate tests that do that :)

Actually, I realized I forgot to add something to disable those tests in Windows in Py2.  It relies on socketpair that wasn't supported natively in Win until 3.5, and IMHO it wouldn't be worth adding one of the various implementations at this point.  Will add that.

Reply:
Checked and the python code is already not windows compatible so not changing anything in that test.

Reply:
If it's required, we should add it to the Debian, configure script etc. Can I suggest taking out the decorators for the moment until we've got this in, and then looking at adding them back in later to simplify the merge, given if the tests are passing it will work fine without the decorators.

Reply:
For posterity, the decorators are now commented out